### ISSUE TITLE
Fail when `Library` directory doesn't exists

### ISSUE THREAD

**Issue by jdvr** [adrianmg/github-pewpew#38]
**Describe the bug**

On Linux OS the `Library` subdirectory doesn't exists by default.

**To Reproduce**

```
❯ ghpew
github-pewpew v1.2.2
Clean up and remove those unnecessary GitHub repositories. Pew pew!

Sign in to GitHub:
Open: https://github.com/login/device
Code: XXXX-XXXX Copied to clipboard!

< --- Follow process on web browser --- >

Error: ENOENT: no such file or directory, mkdir '/home/jdvr/Library/com.adrianmg.github-pewpew'
```

**Expected behavior**
it works

**Screenshots**

** Please complete the following information:**
 - OS: KDE Neon (Ubuntu 20.04)
 - Terminal zsh
 - Version 1.2.2

**Additional context**

**Comment by adrianmg** [comment]
Thanks for reaching out @davidsneighbour! This makes total sense. Do you care to contribute to the open PR?

**Comment by adrianmg** [comment]
Fixed in #37
